tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraEmailSendMethodContract.scala

```diff
@@ -55,154 +78,1613 @@ trait LinagoraEmailSendMethodContract {
+                    |    "sessionState": "${SESSION_STATE.value}",
+                    |    "methodResponses": [
+                    |        [
+                    |            "error",
+                    |            {
+                    |                "type": "invalidArguments",
+                    |                "description": "#notReference cannot be referenced in current method call"
+                    |            },
+                    |            "c1"
+                    |        ]
+                    |    ]
+                    |}""".stripMargin)
   }
 
   @Test
   def implicitSetShouldNotBeAttemptedWhenNotSpecified(server: GuiceJamesServer): Unit = {
+    val request : String =
+      s"""
+         |{
+         |  "using": [
+         |    "urn:ietf:params:jmap:core",
+         |    "urn:ietf:params:jmap:mail",
+         |    "urn:ietf:params:jmap:submission",
+         |    "com:linagora:params:jmap:pgp"
+         |  ],
+         |  "methodCalls": [
+         |    [
+         |      "Email/send",
+         |      {
+         |        "accountId": "$ACCOUNT_ID",
+         |        "create": {
+         |          "K87": {
+         |            "email/create": {
+         |              "mailboxIds": {
+         |                "${getBobInboxId(server).serialize}": true
+         |              },
+         |              "subject": "World domination",
+         |              "htmlBody": [
+         |                {
+         |                  "partId": "a49d",
+         |                  "type": "text/html"
+         |                }
+         |              ],
+         |              "bodyValues": {
+         |                "a49d": {
+         |                  "value": "$HTML_BODY",
+         |                  "isTruncated": false,
+         |                  "isEncodingProblem": false
+         |                }
+         |              }
+         |            },
+         |            "emailSubmission/set": {
+         |              "envelope": {
+         |                "mailFrom": {
+         |                  "email": "${BOB.asString}"
+         |                },
+         |                "rcptTo": [
+         |                  {
+         |                    "email": "${ANDRE.asString}"
+         |                  }
+         |                ]
+         |              }
+         |            }
+         |          }
+         |        }
+         |      },
+         |      "c1"
+         |    ]
+         |  ]
+         |}""".stripMargin
 
+    val response: String = `given`
+      .body(request)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response)
+      .inPath("methodResponses[1]")
+      .isAbsent()
   }
   //endregion
 
-
   //region basic valid request
   @Test
   def methodShouldReturnFailWhenMissingOneCapability(): Unit = {
```

Comment:
```
GIVEN BOB has encryption activated
AND ALICE do not have encryption activated
WHEN BOB sends a message to ALICE
THEN the message is stored encrypted in BOB mailboxes.
```

Reply:
I think it's obvious because it is processed by the Mailbox Manager layer, not in JmapMethod. 

Is it still necessary?

Reply:
Mmm at least to show the message is well stored in Bob mailbox?

Reply:
Yes, it's already 
- emailSendShouldAddAnEmailInTargetMailbox
- emailSendShouldSendMailSuccessfullyToSelf